tests/CommonTests.cs

```diff
@@ -803,5 +871,52 @@ private async Task<LightningInvoice> GetPaidInvoice(ILightningInvoiceListener li
 
             return invoice;
         }
+
+        private static void AssertUnpaid(LightningInvoice invoice, LightMoney expectedAmount = null)
+        {
+            expectedAmount ??= LightMoney.MilliSatoshis(10000);
+            Assert.NotNull(invoice.BOLT11);
+            Assert.Equal(expectedAmount, invoice.Amount);
+            Assert.Null(invoice.PaidAt);
+            Assert.Equal(LightningInvoiceStatus.Unpaid, invoice.Status);
+        }
+
+        private async Task EnsureConnectedToDestinations((string Name, ILightningClient Customer, ILightningClient Merchant) test)
+        {
+            await Task.WhenAll(
+                WaitServersAreUp($"{test.Name} Customer ({(await test.Customer.GetInfo()).NodeInfoList.Select(e => e.NodeId).First()}):", test.Customer),
+                WaitServersAreUp($"{test.Name} Merchant ({(await test.Merchant.GetInfo()).NodeInfoList.Select(e => e.NodeId).First()}):", test.Merchant));
+            Logs.Tester.LogInformation($"{test.Name}: Connecting channels...");
+            
+            var cashcow = await GetRPCClient();
+            var customerNodeClient = test.Customer is LndHubLightningClient ? Tester.CreateLndClient() : test.Customer;
+            var merchantNodeClient = test.Merchant is LndHubLightningClient ? Tester.CreateLndClientDest() : test.Merchant;
```

Comment:
Had to introduce this destinction, because LNDhub does not support the channel related methods. Nevertheless we want to test them, so we use the underlying LND nodes to connect the channels.